src/System.Collections/tests/Generic/List/ForEach_Action.cs

```diff
@@ -107,6 +107,8 @@ public void CallForEach(T[] items, Action<T> action)
             for (int i = 0; i < items.Length; ++i)
                 list.Add(items[i]);
 
+            Assert.Throws<ArgumentNullException>("action", () => list.ForEach(null));
+
```

Comment:
> Is that why there's no List <T> in the future branch?

No, or at least not directly.  The future branch is where we've been adding new APIs that won't be in the initial release, but we do most work in the master branch.  We only merge master to future on-demand when we need to, and as such the master branch is typically many commits ahead of the future branch (other than the commits that only go to future for the new APIs).  This repo has up until recently has only had the code that runs on CoreCLR, but there was a recent push to also add the code that runs on CoreRT, at which point ```List<T>``` was added to this repo.  We haven't had a need to merge master to future since then.